src/client/src/mockData/extensionModules/mockCoreTSModule.ts

```diff
@@ -4,7 +4,7 @@ import * as mockWebData from "./mockData/mockWebPlatformData";
 import * as mockReactNativeData from "./mockData/mockReactNativePlatformData";
 
 //DEV:Change mock data to change platform
-const mockData = mockWebData;
+const mockData = mockReactNativeData;
```

Comment:
better commit this with the Web mock instead?

Reply:
Done.